### PR TITLE
Dev to Main

### PR DESCRIPTION
**Pull request by CBColleg3**

### CODE REVIEW

**Review by mwang840 — COMMENTED** [review]
Looks good, whats with the **startsWith** for the conditionals? Did you forgot a for loop?

File: src/main.ts
Comment (mwang840) [inline src/main.ts:1]: No title "Buccaneer Brawl"?

**Review by mwang840 — APPROVED** [review]

File: src/Classes/player.ts
Comment (mwang840) [inline src/Classes/player.ts:1]: should it be === since it is typescript not java, c++, or python syntax?

Comment (mwang840) [reply src/Classes/player.ts:1]: specifcally line 60 in the else if statement

File: public/assets/audio/What Shall We Do With the Drunken Sailor_ (String Orchestra).mp3
Comment (Izahe) [inline public/assets/audio/What Shall We Do With the Drunken Sailor_ (String Orchestra).mp3:1]: LOL

**Review by Izahe — APPROVED** [review]
Good stuff guys

File: src/Classes/player.ts
Comment (CBColleg3) [reply src/Classes/player.ts:1]: resolved!

**Pull request merged**